python/pyspark/tests.py

```diff
@@ -874,6 +874,15 @@ def test_sortByKey_uses_all_partitions_not_only_first_and_last(self):
             for size in sizes:
                 self.assertGreater(size, 0)
 
+    def test_pipe_functions(self):
+        data = ['1', '2', '3']
+        rdd = self.sc.parallelize(data)
+        with QuietTest(self.sc):
+            self.assertRaises(Py4JJavaError, rdd.pipe('cc').collect)
+        result = rdd.pipe('cat').collect()
```

Comment:
Yes, that's something in my mind. Maybe add a option to `RDD.pipe()` would be friendly, having the default behavior as before.